modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -111,6 +114,10 @@ const loadPlugins = (editor: Editor, suffix: string) => {
     plugin = Tools.trim(plugin);
 
     if (plugin && !PluginManager.urls[plugin]) {
+      if (plugin === 'rtc' && platform.browser.isIE()) {
```

Comment:
Would this be better earlier in the process? As at this stage we've already loaded the theme, setup various event handlers, etc... I'm thinking this might be better in `EditorSettings.processPlugins` for example? or even in the `Editor` constructor.

Reply:
I guess that makes sense I moved it to EditorSettings.ts instead.